src/Dev/Deprecation.php

```diff
@@ -223,29 +256,82 @@ public static function set_enabled($enabled)
         // noop
     }
 
+    /**
+     * If true, any E_USER_DEPRECATED errors should be treated as coming
+     * directly from this class.
+     */
+    public static function triggeringError(): bool
```

Comment:
```suggestion
    public static function isTriggeringError(): bool
```

Reply:
Still need to change method name to slightly nicer version

Reply:
done